### ISSUE TITLE
[A11y] It's very difficult to use the new accessibility checker when there are multiple iframes on the page being tested.

### ISSUE THREAD

**Issue by MarkFalconbridge** [IBMa/equal-access#118]
It's very difficult to use the accessibility checker when there are multiple iframes on the page being tested.

In my case I have a test harness which is not part of the product to be shipped and then hosted within an iframe the product that is shipped.

The DAP tool used to give me the violations and recommendations on an iframe by iframe basis and would group issues within each iframe by severity so it was easy to look at the report for the frame of interest and see whether there were any violations within that frame.

Using the accessibility checker I can select the iframe from the elements panel and that makes the accessibility report jump to any issues for the iframe element but doesn't give me an easy way to look at all the violations in the DOM beneath that iframe.

I'm finding it fairly unusable for this scenario compounded by the fact that in Chrome you have to have the dev tools window docked meaning that everything is very cramped.

**Comment by drjoho** [comment]
@MarkFalconbridge Hi I have a workaround for the detached Accessibility Checker. Before you detach the dev tools perform the scan in the Accessibility Checker (Both the Accessibility Assessment tab and the Accessibility Checker tab). Then detach and the Checker will work for the rest of your session. Hope this helps. Let me know if you have any problems with this and we can do a webex.

**Comment by MarkFalconbridge** [comment]
@drjoho Thanks for the workaround, that certainly gives me a bit more screen real estate to see the scan results.

**Comment by drjoho** [comment]
@MarkFalconbridge if you get the latest version of our checker at the chrome store you won't need the workaround for our checker working in detach dev tools as that issue had been fixed. Now your workflow will work more smoothly.

**Comment by MarkFalconbridge** [comment]
@drjoho That's definitely better.
